### ISSUE TITLE
Short answer issue in Fordson

### ISSUE THREAD

**Issue by mladwig** [dbnschools/moodle-theme_fordson#142]
We recently switched from Boost to Fordson and are running Moodle 3.9. We have an issue with how our short answer questions appear.

In Boost they looked like this.
![Boost-shortanswer](https://user-images.githubusercontent.com/85907843/121965577-16cc4500-cd22-11eb-840b-c48e1f463b32.png)

However, in Fordson it is broken up on new lines like this.
![Fordson-shortanswer](https://user-images.githubusercontent.com/85907843/121965641-2f3c5f80-cd22-11eb-81fc-8a7725a006d6.png)

Is there a way to fix this issue? I tested in on two different Moodle sites with 3.7 and 3.9.

Thanks,
-Matt

**Comment by thebenkahn** [comment]
Hi Matt, it's quite some time later but I ran into this today and am watching the issue. 

I found that the issue appears to be that the input label in Fordson does not get the d-inline class that should be applied (at least that is what happens in Boost where the issue does not occur). Hopefully that is helpful to the developer....

In the meantime you can fix this if you don't mind adding a bit of jQuery to the Additional HTML section under Site Admin -> Appearance to add the class back in.

In my case I added the following to the Before BODY is closed section:

```
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
<script>
$(document).ready(function(){
    $(".path-mod-quiz  .que, .shortanswer input.form-control").addClass("d-inline");
})
</script>
```
